SimpleBrowser/Internal/ObjectExtensions.cs

```diff
@@ -35,13 +34,21 @@ public static string ToQueryString(this object o)
 
 		public static string ToJson(this object obj)
 		{
-			return new JavaScriptSerializer().Serialize(obj);
-		}
+#if NET40
+            return new System.Web.Script.Serialization.JavaScriptSerializer().Serialize(obj);
```

Comment:
@Styxxy In the interest of eliminating a conditional code path, can the .NET Framework use Newtonsoft JSON Nuget package as well?

Reply:
Yes the package supports .NET 4 (I referenced version 9.0.1 : https://www.nuget.org/packages/Newtonsoft.Json/9.0.1 ).

Remark : "breaking" change as we pull in new dependencies. Version shoot definitely be bumped.

Reply:
@Styxxy Why is it a breaking change? Does Newtonsoft serialize differently?

Reply:
No it doesn't, however to the consumer (of the package) it is "breaking", thus a new (major) version should be created.

Reply:
@kevingy I removed the code branching; in this PR now we always depend on Newtonsoft